apps/web/src/pages/settings/organization/OrganizationLogo.tsx

```diff
@@ -0,0 +1,65 @@
+import { errorMessage } from '@novu/design-system';
+import { MimeTypesEnum } from '@novu/shared';
+import React, { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { ProfileImage } from '../../../components/shared';
+
+type FormValues = {
+  logoUrl: string;
+};
+
+const MIME_TYPES = {
+  'image/jpeg': MimeTypesEnum.JPEG,
+  'image/png': MimeTypesEnum.PNG,
+};
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Didn't we extract this somewhere that was shared?‏